go_node_engine/cmd/version.go

```diff
@@ -10,12 +10,12 @@ func init() {
 	rootCmd.AddCommand(versionCmd)
 }
 
-var Version = "None"
+var version = "None"
```

Comment:
This might cause problems. Have you checked if the build script still replaces the version number if we don't export this variable?